media_driver/linux/common/os/i915/include/uapi/i915_drm.h

```diff
@@ -1069,8 +1069,17 @@ struct drm_i915_gem_execbuffer2 {
  * dma fences to wait upon or signal.
  */
 #define I915_EXEC_FENCE_ARRAY   (1<<19)
+/*
```

Comment:
This commit should be substituted with the official pull of the updated i915 uAPI (once it will land to drm-next) according to the instruction: https://github.com/intel/media-driver/blob/master/media_driver/linux/common/os/i915/include/uapi/README.

Reply:
I can update the PR with "$ make headers_install INSTALL_HDR_PATH=/path/to/install" once the uAPI changes landed drm-next.